integration/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.7"
+
+x-restart: &restart_policy
+  restart: "${COMPOSE_RESTART_BEHAVIOR:-on-failure}"
+x-network: &network
+  networks:
+    - internal
+x-default: &default
+  <<: *restart_policy
+  <<: *network
+
+services:
+  sn1:
```

Comment:
it better to have SN1 started finish first, delay some seconds before starting SN2/SN3 to SN1 to be a metadb leader

Reply:
we are not worried about metadb right now